tests/FSharpVSPowerTools.Core.Tests/OutliningTests.fs

```diff
@@ -0,0 +1,144 @@
+﻿#if INTERACTIVE
+#r "../../bin/FSharp.Compiler.Service.dll"
+#r "../../packages/NUnit/lib/nunit.framework.dll"
+#load "../../src/FSharpVSPowerTools.Core/Utils.fs"
+      "../../src/FSharpVSPowerTools.Core/UntypedAstUtils.fs"
+      "../../src/FSharpVSPowerTools.Core/LanguageService.fs"
+      "TestHelpers.fs"
+#else
+module FSharpVSPowerTools.Core.Tests.UntypedAstUtils.Outlining
+#endif
+
+open System.IO
+open NUnit.Framework
+open FSharpVSPowerTools
+open FSharpVSPowerTools.UntypedAstUtils.Outlining
+
+let fileName = Path.Combine (__SOURCE_DIRECTORY__, __SOURCE_FILE__)
+let projectFileName = Path.ChangeExtension(fileName, ".fsproj")
+let sourceFiles = [| fileName |]
+let framework = FSharpCompilerVersion.FSharp_3_1
+let languageService = LanguageService()
+
+let opts source = 
+    let opts = 
+        languageService.GetCheckerOptions (fileName, projectFileName, source, sourceFiles, LanguageServiceTestHelper.args, [||], framework) 
+        |> Async.RunSynchronously
+    { opts with LoadTime = System.DateTime.UtcNow }
+
+let parseSource source = 
+    languageService.ParseFileInProject(opts fileName, fileName, source)
+    |> Async.RunSynchronously
+
+let (=>) (source: string) (lineRanges: (int * int) list) =
+    let ast = (parseSource source).ParseTree
+    match ast with
+    | Some tree ->
+        let multilineRanges =
+            getOutliningRanges tree
+            |> Seq.filter (fun r -> r.StartLine <> r.EndLine)
+            |> Seq.map (fun r -> r.StartLine, r.EndLine)
+            |> List.ofSeq
+        assertEqual lineRanges multilineRanges
+    | None -> failwithf "Expected there to be a parse tree for source:\n%s" ""
```

Comment:
There should be `source` instead of `""`.
I suggest to output the AST in case of failure, like this:
```fsharp
let (=>) (source: string) (lineRanges: (int * int) list) =
    let ast = (parseSource source).ParseTree
    try
        match ast with
        | Some tree ->
            let multilineRanges =
                getOutliningRanges tree
                |> Seq.filter (fun r -> r.StartLine <> r.EndLine)
                |> Seq.map (fun r -> r.StartLine, r.EndLine)
                |> List.ofSeq
            assertEqual lineRanges multilineRanges
        | None -> failwithf "Expected there to be a parse tree for source:\n%s" source
    with _ -> debug "AST:\n%A" ast
```